doc/User Guide/Introduction/Readme.md

```diff
@@ -75,10 +75,10 @@ verdict of its child steps. A verdict has one of 6 severities, detailed in the f
 | 5        | Aborted      | User aborted test plan                                             |
 | 6        | Error        | An error occurred. Check [session logs](#session-logs) for more information |
 
-This means that, for a test plan to output a *Pass* verdict, at least one step must set their verdict to *Pass*, and
-the rest must either set their verdict to *Pass* or *NotSet* verdict. This is also the most typical behavior for parent steps
-containing child steps; a *Sequential* step passes if all of its children pass. However, this is not a rule. Parent
-steps decide their own verdict conditions.
+This hierarchy determines that, for a test plan to output a *Pass* verdict, at least one step must set its verdict to *Pass*, and
+the rest must either set their verdict to *Pass* or *NotSet*. This choice represents the most typical behavior for parent steps
+containing child steps; a *Sequential* step passes if all of its children pass. However, this is not a hard and fast rule: parent
+steps decide overide the verdicts of their children and set their own verdict conditions.
```

Comment:
Weird wording here. Should it say "decide" or "override"?

Reply:
Good catch.  Better to say
. . . parent steps decide whether to override the verdicts of their children . . . 
(insert words "whether to" between "decide" and "override".